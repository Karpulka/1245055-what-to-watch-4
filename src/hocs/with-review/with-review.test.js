import React from "react";
import renderer from "react-test-renderer";
import withReview from "./with-review";

const MockComponent = () => {
  return <div/>;
};

it(`Render withReview`, () => {
  const ReviewWrapper = withReview(MockComponent);

  const tree = renderer
    .create(<ReviewWrapper />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
