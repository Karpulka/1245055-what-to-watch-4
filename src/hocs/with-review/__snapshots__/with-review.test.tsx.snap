// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render withReview 1`] = `<div />`;
